### PR TITLE
50off landing

### PR DESCRIPTION
**Pull request by AWare**
This turned into a bit more work than intended for a landing page, so I've split out the quote changes into their own branch- which I'll drop a WIP for soon.

Roundel: Deployed  as #1060 

**Package Promo:**
-Discount pricing!
image
-A new, clean look we can use!
image
-Darker buttons!

Landing Page:
(in comment)

### CODE REVIEW

File: frontend/assets/stylesheets/components/_packages.scss [hunk 1]
```diff
@@ -169,8 +220,7 @@ $_package-promo__actions_height: 117px;
         }
         .package-promo__actions {
             position: static;
-            padding-top: rem($gs-gutter / 4);
-            padding-bottom: rem($gs-gutter / 4);
+            padding: rem($gs-gutter / 4);
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_packages.scss]: you can remove these `rem()` functions from any code you touch, we have the px-to-rem plugin now

File: frontend/assets/stylesheets/components/_pricing.scss [hunk 2]
```diff
@@ -59,6 +60,28 @@
         margin-top: -3px;
     }
 
+    .price-info__value_discount{
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_pricing.scss]: should be `value-discount` by our BEM conventions

File: frontend/app/views/fragments/tier/packagePromo.scala.html [hunk 3]
```diff
@@ -51,19 +54,35 @@
...
+                    case PaidPlans(p) => {
+                        @fragments.pricing.paidPriceInfo(p, cg.currency)
+                    }
+                }
+                </div>
+                <a class="action qa-package-@plans.tier.slug u-fullwidth  @if(!secondaryButton.isDefined){u-no-bottom-margin}" href="@promoButton.to" @promoButton.attributes.html>
+                    @promoButton.text
+                </a>
+                @for(button <- secondaryButton){
+                    <br/>
```
Comment (joelochlann) [inline frontend/app/views/fragments/tier/packagePromo.scala.html]: avoid using `<br />` tags just to space sections. https://developer.mozilla.org/en/docs/Web/HTML/Element/br

File: frontend/app/views/promotions/discountOffer.scala.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+@(partnerPlans: PaidMembershipPlans[Current, Partner],
+    pageInfo: PageInfo,
+    pageImages: Seq[model.ResponsiveImageGroup])(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
+
+@import views.support.Asset
+
+
+@main("Save the guardian, save the world!", pageInfo=pageInfo, countryGroup=Some(countryGroup)) {
+
+    @fragments.common.roundel(classModifier = Some("alt")){Join <br/> as an annual Partner Member before 30 April 2016 and save<br/> £50}()
```
Comment (joelochlann) [inline frontend/app/views/promotions/discountOffer.scala.html]: avoid `<br />`

**Comment by joelochlann** [comment]
Looks OK so far. Just a few comments. As discussed, feel free to use flexbox for stuff (i.e. equal width buttons) if it degrades gracefully.

Also please try and make PRs intelligible to someone coming at them with no context. Clear "before", "afters", etc. Cheers

**Comment by AWare** [comment]
Thanks @joelochlann, sorry for the slightly incomprehensible pr. Updating

**Comment by AWare** [comment]
Screenshots of landing page:
screen shot 2016-03-22 at 14 20 37
screen shot 2016-03-22 at 14 23 55

**Comment by AWare** [comment]
No changes to the width of buttons anymore: 
image

File: frontend/app/views/fragments/promos/discountedPrice.scala.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+@(old: String, offer: String, term: String)
+<div class="price-info">
+    <div class="price-info__discount">
+        <strong class="price-info__value-discount price-info__strikeout">@old</strong>
+    </div>
+    <div class="price-info__discount">
+        <strong class="price-info__value-discount">@offer</strong><span class="price-info__date">/@term</span></div>
```
Comment (joelochlann) [inline frontend/app/views/fragments/promos/discountedPrice.scala.html]: what's the closing `</div>` doing on the same line?

File: frontend/app/views/fragments/tier/packagePromo.scala.html [hunk 6]
```diff
@@ -51,19 +54,34 @@
...
+                }
+                </div>
+                <a class="action qa-package-@plans.tier.slug  @if(!secondaryButton.isDefined){u-no-bottom-margin}" href="@promoButton.to" @promoButton.attributes.html>
+                    @promoButton.text
+                </a>
+                @for(button <- secondaryButton){
+
+                    <a class="action action--no-icon action--secondary " href="@button.to" @button.attributes.html>
+                        @button.text
+                    <svg class="icon-inline action__icon action__icon--right">
```
Comment (joelochlann) [inline frontend/app/views/fragments/tier/packagePromo.scala.html]: indentation of `<svg></svg>`

File: frontend/assets/stylesheets/components/_pricing.scss [hunk 7]
```diff
@@ -16,22 +16,22 @@
     }
 
     @include mq(desktop) {
-        min-height: rem($gs-gutter * 3);
+        min-height: $gs-gutter * 3;
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_pricing.scss]: Cheers for removing `rem()` :gem:

File: frontend/assets/stylesheets/components/_packages.scss [hunk 8]
```diff
@@ -109,6 +115,68 @@ $_package-promo__actions_height: 117px;
...
+        border: none;
+        background: $white;
+        .package-promo__actions {
+            vertical-align: middle;
+
+        }
+    }
+
+    .package-promo__content {
+        display: flex;
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_packages.scss]: The "mobile-first" convention is to define styles for smaller devices first and override with media queries for larger devices, e.g. here a mobile style outside the media query, and then `@include mq($from: phablet)` with the phablet & up specific styles

File: frontend/assets/stylesheets/components/_packages.scss [hunk 9]
```diff
@@ -109,6 +115,68 @@ $_package-promo__actions_height: 117px;
...
+            flex-direction: column;
+        }
+    }
+
+    .package-promo__header {
+        display: none;
+    }
+    .package-promo__actions__pricing {
+        border: none;
+        @include mq($from: mobile, $until: tablet) {
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_packages.scss]: empty media queries

File: frontend/assets/stylesheets/components/_packages.scss [hunk 10]
```diff
@@ -109,6 +115,66 @@ $_package-promo__actions_height: 117px;
     }
 }
 
+//NEW
```
Comment (joelochlann) [inline frontend/assets/stylesheets/components/_packages.scss]: this comment will quickly be out of date :stuck_out_tongue_winking_eye:

**Comment by joelochlann** [comment]
:+1:

**Pull request merged**